ocpi-endpoints-common/src/main/scala/com/thenewmotion/ocpi/common/AuthorizationRejectionHandler.scala

```diff
@@ -0,0 +1,30 @@
+package com.thenewmotion.ocpi.common
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import akka.http.scaladsl.server.directives.BasicDirectives
+import akka.http.scaladsl.server.directives.MiscDirectives
+import com.thenewmotion.ocpi.msgs
+
+// TODO: Could this move to OcpiRejectionHandler?
```

Comment:
OcpiRejectionHandler is actually part of the endpoints-toplevel package, but I think this is wrong and it should be moved to encdpoints-common and then the two can be merged... what do you think @poohsen?